patches/openwrt/0007-fixup-kernel-bridge-Implement-MLD-Querier-wake-up-calls-Android-bug-workaround.patch

```diff
@@ -0,0 +1,78 @@
+From: David Bauer <mail@david-bauer.net>
+Date: Sun, 24 Sep 2023 22:31:50 +0200
+Subject: fixup! kernel: bridge: Implement MLD Querier wake-up calls / Android bug workaround
+
+diff --git a/package/network/config/netifd/patches/0001-bridge-Add-multicast_wakeupcall-option.patch b/package/network/config/netifd/patches/0001-bridge-Add-multicast_wakeupcall-option.patch
+index aeb2758dc9396871b448bbd8d264e143b686b8c7..5f05cccd63496d5c5319297092ac115e287c8a80 100644
+--- a/package/network/config/netifd/patches/0001-bridge-Add-multicast_wakeupcall-option.patch
++++ b/package/network/config/netifd/patches/0001-bridge-Add-multicast_wakeupcall-option.patch
+@@ -90,7 +90,7 @@ Signed-off-by: Linus Lüssing <linus.luessing@c0d3.blue>
+  	DEV_OPT_RXPAUSE			= (1ULL << 34),
+  	DEV_OPT_TXPAUSE			= (1ULL << 35),
+  	DEV_OPT_AUTONEG			= (1ULL << 36),
+-+	DEV_OPT_MULTICAST_WAKEUPCALL	= (1ULL << 63),
+++	DEV_OPT_MULTICAST_WAKEUPCALL	= (1ULL << 63)
+  };
+  
+  /* events broadcasted to all users of a device */
+@@ -140,3 +140,60 @@ Signed-off-by: Linus Lüssing <linus.luessing@c0d3.blue>
+  }
+  
+  int system_bridge_addif(struct device *bridge, struct device *dev)
++--- a/interface.c
+++++ b/interface.c
++@@ -1248,7 +1248,7 @@ interface_device_config_changed(struct i
++ 	struct blob_attr *ntb[__DEV_ATTR_MAX];
++ 	struct blob_attr *otb[__DEV_ATTR_MAX];
++ 	struct device *dev = if_old->main_dev.dev;
++-	unsigned long diff = 0;
+++	unsigned long diff[2] = {};
++ 
++ 	BUILD_BUG_ON(sizeof(diff) < __DEV_ATTR_MAX / 8);
++ 
++@@ -1267,8 +1267,8 @@ interface_device_config_changed(struct i
++ 	blobmsg_parse(device_attr_list.params, __DEV_ATTR_MAX, ntb,
++ 		blob_data(if_new->config), blob_len(if_new->config));
++ 
++-	uci_blob_diff(ntb, otb, &device_attr_list, &diff);
++-	return diff;
+++	uci_blob_diff(ntb, otb, &device_attr_list, diff);
+++	return !!(diff[0] | diff[1]);
```

Comment:
The `!!` can be omitted, the function already returns `bool`.

I'm not entirely convinced that `|` is used instead of `||` when the result is only used as `bool`, but at least it's consistent with the other place in netifd where a diff of length 2 is checked...